### DISCUSSION TITLE
Stuck on bootloop

### DISCUSSION THREAD

**Discussion by RandomDude101Offical** [CosmosOS/Cosmos#3105] [Q&A] [answered]
![Screenshot 2024-11-20 200154](https://github.com/user-attachments/assets/7f66bae1-23ba-46a2-88ab-970c4fbad196)
its stuck on the "Waiting for debugger connection" screen.

**Comment by Gabolate** [comment, marked as answer]
Disable Cosmos Debugger on Project Settings

**Reply by RandomDude101Offical** [reply]
It worked, Thank you.

**Reply by Gabolate** [reply]
No problem 👍
